force-app/main/default/flows/Contract_Creation.flow-meta.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
try to sum up the task in more short manner. what if we combine both condition in one element. you are setting the end date according to the opportunity's start date try to set this end date exact one year later from the day on which the stage is updated to closed won.

Reply:
Done